templates/taxi/driver_detail.html

```diff
@@ -9,6 +9,8 @@ <h1>
   <p><strong>Last name:</strong> {{ driver.last_name }}</p>
   <p><strong>License number:</strong> {{ driver.license_number }}</p>
   <p><strong>Is staff:</strong> {{ driver.is_staff }}</p>
+  <p><a href="{% url "taxi:driver-update" pk=driver.id %}">Update</a></p>
+  <p><a href={% url "taxi:driver-delete" pk=driver.id %}>Delete</a></p>
```

Comment:
The URL tag for `driver-delete` should be enclosed in quotes. It should be `<a href="{% url 'taxi:driver-delete' pk=driver.id %}">Delete</a>` to ensure proper URL resolution.